packages/commerce-sdk-react/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@salesforce/commerce-sdk-react",
   "version": "1.0.1",
```

Comment:
This package version should be `1.0.2-preview.1` → that's probably why CI fails when `npm ci` this package and complains that it cannot find the lock file. Because the `version` is mismatched between package and package lock.